cafebabel/templates/articles/macros.html

```diff
@@ -66,7 +66,7 @@
     <h3><a href={{ article.detail_url }}>{{ article.title }}</a></h3>
     <p>
       <span>Published on <time pubdate="{{ article.creation_date.date() }}">{{ article.creation_date.date() }}</time></span>
-      — by {{ article.author }}
+      — by <a href={{ url_for('users.detail', id=article.author.pk) }}>{{ article.author }}</a>
```

Comment:
🎉

Reply:
Should we update the tests?

Reply:
no need, right @davidbgk ?

Reply:
I added it with https://github.com/cafebabel/cafebabel.com/pull/245/commits/d0691ba3847c914efb83f6f795c3ef4bd20b84dc